akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/PresPageDAO.scala

```diff
@@ -47,4 +61,51 @@ object PresPageDAO {
         case Failure(e)            => DatabaseConnection.logger.debug(s"Error updating current on PresPage: $e")
       }
   }
+
+  def resizeAndMovePage(presentation: PresentationPage, presentationId: String) = {
+    DatabaseConnection.db.run(
+      sqlu"""UPDATE pres_page SET
```

Comment:
Use `sqlu` for updates only for edge cases that requires it!
For normal update use the default Slick function for update, for instance:

```
  def update(regUser: RegisteredUser) = {
    DatabaseConnection.db.run(
      TableQuery[UserDbTableDef]
        .filter(_.userId === regUser.id)
        .map(u => (u.guest, u.guestStatus, u.role, u.authed, u.joined, u.banned, u.loggedOut))
        .update((regUser.guest, regUser.guestStatus, regUser.role, regUser.authed, regUser.joined, regUser.banned, regUser.loggedOut))
    ).onComplete {
        case Success(rowsAffected) => DatabaseConnection.logger.debug(s"$rowsAffected row(s) updated on user table!")
        case Failure(e)            => DatabaseConnection.logger.debug(s"Error updating user: $e")
      }
  }
```